djangocms_pageadmin/admin.py

```diff
@@ -26,7 +26,7 @@
 
 
 class PageContentAdmin(VersioningAdminMixin, DefaultPageContentAdmin):
-    change_list_template = "admin/change_list.html"
+    change_list_template = "admin/pageadmin_change_list.html"
```

Comment:
"admin/djangocms_pageadmin/pagecontent/change_list.html" then.

Reply:
I don't understand? It's working as is.

Reply:
I tried that but couldn't get it to work. For some reason it doesn't actually load the override.

Reply:
Did you also move the file there?

Reply:
Yes

Reply:
Well, try to make it work. This won't pass the review.

Reply:
Why won't it pass review?

Reply:
I don't think what you're asking for is possible. That only works if there is a model for the changelist, but there's no model, so we have to use a custom template. Monika agrees.